<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisWindow</class>
 <widget class="QMainWindow" name="RegisWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>500</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>RegisWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;注 册&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>注 册</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="regConfirm">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regCancel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>280</width>
      <height>187</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;黑体&quot;;
font: 12pt &quot;Adobe Heiti Std&quot;;</string>
       </property>
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;黑体&quot;;
font: 12pt &quot;Adobe Heiti Std&quot;;</string>
       </property>
       <property name="text">
        <string>手机号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;黑体&quot;;
font: 12pt &quot;Adobe Heiti Std&quot;;</string>
       </property>
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="address"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="phone"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;黑体&quot;;
font: 12pt &quot;Adobe Heiti Std&quot;;</string>
       </property>
       <property name="text">
        <string>住址：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="mail"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="password_2"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;黑体&quot;;
font: 12pt &quot;Adobe Heiti Std&quot;;</string>
       </property>
       <property name="text">
        <string>确认密码：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;黑体&quot;;
font: 12pt &quot;Adobe Heiti Std&quot;;</string>
       </property>
       <property name="text">
        <string>邮箱：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="password"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="hint">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[提示信息]</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
